index.js

```diff
@@ -267,4 +263,13 @@ Emailer.admin = {
     }
 };
 
+Emailer.addMiddleware = function(data, callback) {
+    data.app.head('/emailer-mandrill/reply', function(req, res) {
+        res.sendStatus(200);
+    });
+    data.app.post('/emailer-mandrill/reply', Emailer.receive);
+
+    callback();
+};
+
```

Comment:
I don't understand... the two routes as they are don't do anything related to user (well, except for the actual email parsing)... and the logic itself is identical. Unless it wasn't working in `static:app.route`, there should be no reason to move it out of there...